### PR TITLE
[com_actionlogs] namespaced

### PR DESCRIPTION
**Pull request by alikon**
### Summary of Changes
use namespaced classes

### Testing Instructions
code review
com_actionlogs works as before

### CODE REVIEW

File: administrator/components/com_actionlogs/actionlogs.php [hunk 1]
```diff
@@ -9,11 +9,17 @@
 
 defined('_JEXEC') or die;
 
-if (!JFactory::getUser()->authorise('core.admin'))
+use Joomla\CMS\Access\Exception\NotAllowed;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\MVC\Controller\BaseController;
+
+
```
Comment (Quy) [inline administrator/components/com_actionlogs/actionlogs.php]:
```suggestion
```

File: administrator/components/com_actionlogs/models/fields/logtype.php [hunk 2]
```diff
@@ -10,8 +10,13 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\Application\ApplicationHelper;
+use Joomla\CMS\Factory;
+use Joomla\CMS\Form\FormHelper;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
 
-JFormHelper::loadFieldClass('checkboxes');
+
```
Comment (Quy) [inline administrator/components/com_actionlogs/models/fields/logtype.php]:
```suggestion
```

File: administrator/components/com_actionlogs/models/actionlog.php [hunk 3]
```diff
@@ -140,13 +143,13 @@ protected function sendNotificationEmails($messages, $username, $context)
 			return;
 		}
 
-		$layout    = new JLayoutFile('components.com_actionlogs.layouts.logstable', JPATH_ADMINISTRATOR);
+		$layout    = new LayoutFile('components.com_actionlogs.layouts.logstable', JPATH_ADMINISTRATOR);
```
Comment (Quy) [inline administrator/components/com_actionlogs/models/actionlog.php]:
```suggestion
		$layout    = new FileLayout('components.com_actionlogs.layouts.logstable', JPATH_ADMINISTRATOR);
```

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 3b1b90eb32c47976dc040808a6c56e4453f8386d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23341">issues.joomla.org/tracker/joomla-cms/23341</a>.</sub>

File: administrator/components/com_actionlogs/models/fields/extension.php [hunk 4]
```diff
@@ -59,7 +63,7 @@ public function getOptions()
 			foreach ($extensions as $extension)
 			{
 				ActionlogsHelper::loadTranslationFiles($extension);
-				$options[] = JHtml::_('select.option', $extension, JText::_($extension));
+				$options[] = HTMLHelper::_('select.option', $extension, JText::_($extension));
```
Comment (Quy) [inline administrator/components/com_actionlogs/models/fields/extension.php]: Namespace `JText`

Comment (alikon) [reply administrator/components/com_actionlogs/models/fields/extension.php]: done

Comment (Quy) [reply administrator/components/com_actionlogs/models/fields/extension.php]: Also add the use statement.

Comment (alikon) [reply administrator/components/com_actionlogs/models/fields/extension.php]: :face_with_head_bandage:

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 683ff08e0240388d71b7d2fa7a347282ee270361<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23341">issues.joomla.org/tracker/joomla-cms/23341</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 683ff08e0240388d71b7d2fa7a347282ee270361<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23341">issues.joomla.org/tracker/joomla-cms/23341</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23341">issues.joomla.org/tracker/joomla-cms/23341</a>.</sub>

**Pull request merged**